stitch/options.py

```diff
@@ -0,0 +1,48 @@
+from collections.abc import Mapping
+from functools import wraps
+
+def as_bool():
```

Comment:
😄  I noticed that halfway through. I'll probably end up using it, but with custom coercion rules to unwrap the pandoc JSON.